### ISSUE TITLE
(Regression in 0.2.3) "cover/lcov.info": no such file or directory

### ISSUE THREAD

**Issue by skyqrose** [dariodf/lcov_ex#12]
With a dependabot upgrade for the new 0.2.3 version, my project has a new failure in CI:

```
Adding to lcov file... 
** (File.Error) could not write to file "cover/lcov.info": no such file or directory
    (elixir 1.13.4) lib/file.ex:1054: File.write!/3
    (lcov_ex 0.2.3) lib/lcov_ex.ex:41: anonymous fn/3 in LcovEx.start/2
    (mix 1.13.4) lib/mix/tasks/test.ex:538: Mix.Tasks.Test.do_run/3
    (mix 1.13.4) lib/mix/task.ex:397: anonymous fn/3 in Mix.Task.run_task/3
    (mix 1.13.4) lib/mix/task.ex:451: Mix.Task.run_alias/5
    (mix 1.13.4) lib/mix/cli.ex:84: Mix.CLI.run_task/2
    (elixir 1.13.4) lib/code.ex:1183: Code.require_file/2
Error: Process completed with exit code 1.
```

Unfortunately it's closed source so I can't link it directly here. It's not an umbrella app, and it is working with v0.2.2.

I believe it could be caused by [changing "write" to "append"](https://github.com/dariodf/lcov_ex/commit/35fd6dbdea42e2dc9c069022bfbe6e11187566f1#diff-9090fc8d40f90a53193ec7570379b7a7f9f9c2161657d830a8644f4f54554a5bR41)

(Side note: If this repo had github releases or tags or a changelog for each version, that could have helped me investigate this a little faster.)

**Comment by dariodf** [comment]
Hi! I have a fix for it in #10. Would you mind taking a look?

**Comment by dariodf** [comment]
Hi @skyqrose. This issue is already fixed in [v0.2.4](https://github.com/dariodf/lcov_ex/releases/tag/v0.2.4). Thanks for the report!
